app/helpers/react_components/student/solution_summary.rb

```diff
@@ -5,13 +5,25 @@ class SolutionSummary < ReactComponent
 
       def to_s
         super("student-solution-summary", {
-          iteration: SerializeIteration.(iteration),
+          solution_id: iteration.solution.uuid,
+          request: request,
           is_practice_exercise: iteration.exercise.practice_exercise?,
           links: links
         })
       end
 
       private
+      def request
+        {
+          endpoint: Exercism::Routes.temp_solution_url(iteration.solution.uuid),
```

Comment:
@iHiD This is where to change the endpoint.

We prepopulate the query with data from the server too, so this component won't fetch the data on initial load.

Reply:
Great.

Reply:
`api_solution_path(solution.uuid, sideloads: [:latest_iteration])`